### DISCUSSION TITLE
PV18 5248 works only when A/B wires connected to RJ45 port

### DISCUSSION THREAD

**Discussion by Rogero-** [vladyspavlov/esphome-must-inverter#11] [General]
hello,

just sharing my experience with my second inverter Must Pro PV18 5248, my first inverter (also PV18 5248) worked fine when i used the USB port with the ESP8266 to get the 5v power and the A/B RS485 data wires, so i built another identical dongle, same hardware, same wiring and i couldn't get it to work on my second inverter, i changed the USB extension, changed the ESP8266 board, always getting "unknown" values from the inverter, then i tried wiring an ethernet cable to the RJ45 port of the inverter and it worked right away.

i'm sharing this to tell others that the USB port connection for A/B may not always work, in this case try the RJ45 port.

Cheers.

**Comment by goblinmaks** [comment]
Did you use same esp+rs485 board or identical one ?

**Reply by Rogero-** [reply]
identical ones, same model of ESP8266 and rs485 board with Flow control pins, all the details are here --> https://github.com/vladyspavlov/esphome-must-inverter/discussions/9

Also i tried the first working dongle i built on the second inverter and it didn't work, same result "Unknown" values, Second dongle tested on First inverter from USB port and it worked fine.

So the second inverter only works when A/B are connected to the RJ45 port .
